package.json

```diff
@@ -40,6 +42,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "test:ci": "react-scripts test --coverage --testResultsProcessor ./node_modules/jest-sonar-reporter",
+    "mesa:test:ci": "react-scripts test --coverage --testResultsProcessor ./node_modules/jest-sonar-reporter --watchAll=false",
```

Comment:
Probably not the best practice, if you want you can run this same command by using:

```
npm run test_ci — —watchAll=false
```

Reply:
Those are double dash’s btw, github formats it weird